scripts/check_type.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/sh
 
-if ! (mypy --ignore-missing-imports \
-           --disallow-untyped-defs \
-           --disable-error-code override peekingduck); then
+if ! (mypy --show-error-codes peekingduck); then
```

Comment:
Yep, this script and the script for linting will be deprecated. It is currently [listed as an issue](https://github.com/aimakerspace/PeekingDuck/issues/360) to be handled in the future. This is because I preferred to keep this work to a separate thread and focus on the changes to enhance the custom configurations.